docs/README.md

```diff
@@ -1,503 +1,491 @@
+|                    | `mode`                   | No            | Mode                              | The checkout mode for the shopper. By default, the mode is set to `payment`. Set it to `recurring` if you want to save the payment method for future payments with the shopper. Supported payment methods include Alipay, AlipayHK, Gcash, Dana, KakaoPay, Touch ‘n Go. Only merchant-initiated subsequent payments are supported.    |
+|                    | `client_secret`          | No            | string                            | The client\_secret of the Customer object. It's required when `mode` is set to `recurring` else optional.     |
+|                    | `customer_id`            | No            | string                            | The id of the Customer for whom the PaymentIntent or PaymentConsent is created – the id returned in [Create a Customer](https://www.airwallex.com/docs/api#/Payment_Acceptance/Customers/_api_v1_pa_customers_create/post "https://www.airwallex.com/docs/api#/Payment_Acceptance/Customers/_api_v1_pa_customers_create/post") API response |
+| `qrcode`           | `intent`                 | Yes           | Intent                            | The `id` and `client_secret` of the PaymentIntent you want to checkout.  |
+|                    | `qrcodeMethod`           | Yes           | PaymentMethodWithQrcode\[ \]      | The payment method you want to redirect the shopper to. For example, `alipayhk` , `wechatpay`, `kakaopay`, etc.    |
+| `wechat`           | `intent`                 | Yes           | Intent                            | The `id` and `client_secret` of the PaymentIntent you want to checkout.  |
+| `applePayButton`   | `intent`                 | Yes           | Intent                            | The `id` and `client_secret` of the PaymentIntent you want to checkout. |
+|                    | `amount`                 | Yes           | {value: number; currency:string;} | The amount and currency of the PaymentIntent.    |
+|                    | `country_code`           | Yes           | string                            | Your two-letter ISO 3166 country code.    |
+|                    | `autoCapture`            | No            | boolean                           | Indicates whether the amount must be automatically captured after successful authorization. By default, the value from Create a PaymentIntent API will be used.   |
+|                    | `ApplePayRequestOptions` | No            | ApplePayRequestOptions            | Additional customization options for the button style and configuration for payment processing capabilities, billing and shipping information.    |
```

Comment:
can you help make up all the ApplePayRequestOptions here,  just copy the following and remove this line

| `buttonType`                   | No            | string                                 | The type of button you want to display on your payment form, for example, 'donate' https://developer.apple.com/documentation/apple\_pay\_on\_the\_web/displaying\_apple\_pay\_buttons\_using\_css |
| `buttonColor`                  | No            | string                                 | The color of the button. Default value is 'black'                                                                                                                                                 |
| `totalPriceLabel`              | No            | string                                 | The business name your shoppers will see when they look for the charge on their bank or credit card statement. For example, "COMPANY, INC."                                                       |
| `lineItems`                    | No            | ApplePayJS.ApplePayLineItem\[\];       | A set of line items that explain recurring payments and/or additional charges.                                                                                                                    |
| `billingContact`               | No            | ApplePayJS.ApplePayPaymentContact      | The billing contact information for the shopper.                                                                                                                                                  |
| `requiredBillingContactFields` | No            | ApplePayJS.ApplePayContactField\[\];   | The billing information that you require from the shopper to process the transaction.                                                                                                             |
| `shippingContact`              | No            | ApplePayJS.ApplePayPaymentContact      | Shipping contact information for the user.                                                                                                                                                        |
| `shippingMethods`              | No            | ApplePayJS.ApplePayShippingMethod\[\]; | The list of shipping methods available for a payment request.                                                                                                                                     |
| `shippingType`                 | No            | ApplePayJS.ApplePayShippingType;       | An optional value that indicates how purchased items are to be shipped.                                                                                                                           |
| `supportedCountries`           | No            | string\[ \]                            | A list of two-letter ISO 3166 country codes, used to limit payments to cards from specific countries or regions.                                                                                  |
| `applicationData`              | No            | string                                 | Optional user-defined data.

Reply:
Updated.